frontend/packages/ceph-storage-plugin/src/selectors/index.ts

```diff
@@ -81,3 +81,15 @@ export const calcPVsCapacity = (pvs: K8sResourceKind[]): number =>
 
 export const getSCAvailablePVs = (pvsData: K8sResourceKind[], sc: string): K8sResourceKind[] =>
   pvsData.filter((pv) => getPVStorageClass(pv) === sc && pv.status.phase === status.AVAILABLE);
+
+export const getSCNamesUsingPool = (
+  scData: ListKind<K8sResourceKind>,
+  poolName: string,
+): string[] =>
+  scData.items?.reduce((scNames, sc) => {
+    if (_.get(sc, 'parameters.pool') === poolName) scNames.push(sc.metadata?.name);
+    return scNames;
+  }, []);
+
+export const getPvcNames = (pvcResources: ListKind<K8sResourceKind>): string[] =>
+  pvcResources?.items?.map((pvcConfig) => getStorageClassName(pvcConfig));
```

Comment:
Its giving you sc names and not pvc names.
No need to create a new utility just use it directly in the component as follows:
```suggestion
 const scNames = pvcResources?.items?.map(getStorageClassName);
```